ViewControllers/MostPopularCollectionViewController.swift

```diff
@@ -192,7 +187,7 @@ class MostPopularViewController: UIViewController, UICollectionViewDelegate {
 //        }
     }
 
-    func configureWithData(index: Int) {
+    private func configureWithData(index: Int) {
         let movieName = movies[index].title
         let reviewsScore = "\(movies[index].voteAverage)%"
         let movieDescription = movies[index].overview
```

Comment:
Just a quick tip, instead of accessing the array at index on every line you can do it in a single property and reuse it so you avoid miss typing the index. Also, improves readability
```suggestion
        let movie = movies[index]
        let movieName = movie.title
        let reviewsScore = "\(movie.voteAverage)%"
        let movieDescription = movie.overview
```

Reply:
I do lack consistency, since I did it in some places and didn't in others